src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -78,7 +78,15 @@ std::optional<GUID> CascadiaSettings::FindGuid(const std::wstring& profileName)
     {
         if (profileName == profile.GetName())
         {
-            profileGuid = profile.GetGuid();
+            try
+            {
+                profileGuid = profile.GetGuid();
+            }
+            catch (...)
+            {
+                LOG_CAUGHT_EXCEPTION();
+            }

```

Comment:
You should be able to replace this with `CATCH_LOG();` I think.

Reply:
That makes sense. I thought something was off with this.